src/util/isURL.ts

```diff
@@ -1,9 +1,13 @@
 import isEmail from './isEmail'
+import stripTags from './stripTags'
 
 const REGEX_URL =
   /^(?:http(s)?:\/\/)?(www\.)?[a-zA-Z@:%_\\+~#=]+[-\w@:%_\\+~#=.]*[\w@:%_\\+~#=]+[.:][\w()]{2,6}((\/[\w-()@:%_\\+~#?&=.]*)*)$/i
 
 /** Checks if a string is a URL. */
-const isURL = (s: string) => REGEX_URL.test(s) && !isEmail(s)
+const isURL = (s: string) => {
+  const strippedValue = stripTags(s)
+  return REGEX_URL.test(strippedValue) && !isEmail(strippedValue)
+}
```

Comment:
I'm a little skeptical of `isURL` automatically stripping out the tags, rather than stripping the tags before passing them to `isURL`. If I were to import and use `isURL` elsewhere, I would expect it to only check if the string is a URL, and not necessarily also return `true` if it's a URL enclosed in tags.

However, at the moment, it looks like this utility is only used in places where it would make sense to do this. I don't think this is a blocking issue, but I just wanted to point it out.